sensor_height: 0.83
iteration_number: 10
lpr_number: 50
seeds_threshold: 0.1
distance_threshold: 0.3
# x_range: 3
y_range: 6
output_topic: /ground_points
